Exception Handling/Task3/UserTaskService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using Task3.DoNotChange;
+
+namespace Task3
+{
+    public class UserTaskService
+    {
+        private readonly IUserDao _userDao;
+
+        public UserTaskService(IUserDao userDao)
+        {
+            _userDao = userDao;
+        }
+
+        public void AddTaskForUser(int userId, UserTask task)
+        {
+            if (userId < 0)
```

Comment:
Usually, default values should be better replaced with the following statement:
```suggestion
            if (userId < default)
```

It allows you to avoid extra code changes if your type you validating has updated